src/app/private/scripts/controllers/announcement/createAnnouncementController.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
-angular.module('playerApp').controller('createAnnouncementCtrl', ['$rootScope', '$scope', '$timeout', 'config', 'toasterService', 'announcementService', '$filter', 'uuid4',
-  function ($rootScope, $scope, $timeout, config, toasterService, announcementService, $filter, uuid4) {
+angular.module('playerApp').controller('createAnnouncementCtrl', ['$rootScope', '$scope', '$timeout', 'config', 'toasterService', 'announcementService', '$filter', 'uuid4', 'fileUpload',
+  function ($rootScope, $scope, $timeout, config, toasterService, announcementService, $filter, uuid4, fileUpload) {
```

Comment:
$services should be placed before others, please move $filter.

Reply:
@kochhar resolved
All '$' item should be injected first. cc: @harishgilimi